application/controllers/ReportController.php

```diff
@@ -49,14 +51,29 @@ public function indexAction()
     {
         $this->addTitleTab($this->report->getName());
 
+        $this->controls->getAttributes()->add('class', 'default-layout');
         $this->addControl($this->assembleActions());
 
+        $contentId = $this->content->getAttributes()->get('id')->getValue();
+        $this->sendExtraUpdates([
+            $contentId => Url::fromPath('reporting/report/content', ['id' => $this->report->getId()])
+        ]);
+
+        // Will be replaced once the report content is rendered
+        $this->addContent(new HtmlElement('p'));
+    }
+
+    public function contentAction()
+    {
         Environment::raiseExecutionTime();
         Environment::raiseMemoryLimit();
 
+        $this->view->compact = true;
+        $this->_helper->layout()->disableLayout();
+
         try {
-            $this->addContent($this->report->toHtml());
-        } catch (\Exception $e) {
+            $this->getDocument()->addHtml($this->report->toHtml());
+        } catch (Exception $e) {
             $this->addContent(Error::show($e));
```

Comment:
An error is also shown with tabs I suppose? Should also be added to the document directly.